src/main/ruby/truffleruby/core/match_data.rb

```diff
@@ -153,6 +163,20 @@ def match_length(n)
   def to_s
     self[0]
   end
+
+  private
+
+  def backref_from_arg(index)
+    if Primitive.is_a?(index, String) || Primitive.is_a?(index, Symbol)
+      names_to_backref = Hash[Primitive.regexp_names(self.regexp)]
+      unless names_to_backref.key?(index.to_sym)
```

Comment:
I'm finding this rather difficult to read. There's a lot going on. Can physically separate the `unless` statement block from the lines above and below with vertical space?

Reply:
I rewrote it a bit, does that improve readability?